file.go

```diff
@@ -612,11 +690,14 @@ func growFile(f *File, opts Options) error {
 // allowed to fail. Excessive pages are freed in future transactions anyways.
 // The file is not truncated yet, as the last 2 transaction must agree on the
 // actual file size before truncating. Truncation is postponed to later transactions.
-func shrinkFile(f *File, opts Options) error {
+func shrinkFile(f *File, opts Options) reason {
+	const op = "txfile/shrink"
```

Comment:
Instead of defining all these const locally would it make sense to have them together in one table and only used there? Would make it easier to see what `ops` are there.

Reply:
Idea of `op` is to have something a programmer can grep for and immediately find the position of the function. Introducing another file with definitions will add some more indirection.

The `op` should follow the function name. That's also why there is a newline after `op`. My hope is `op` will be updated once the function name changes.

Also `op` should not occur twice (even when using a const). If we really make this a more general pattern, some linter checking for duplicates would be helpful.